common/source/docs/common-msp-osd-overview-4.2.rst

```diff
@@ -143,7 +155,33 @@ When the OSD switches to this screen it will check the value of the :ref:`OSD2_S
  - OSDn_GSPEED will display max ground speed (or airspeed if ``OSDn_ASPEED_EN`` is set to 1)
  - OSDn_HOMEDIST will alternates max distance from home and total traveled distance every 2 seconds
  - OSDn_RSSI will display min rssi
- 
+
+DisplayPort OSD
+===============
+
+FatShark's SharkByte using ArduPilot custom fonts
+
+.. image:: ../../../images/msp_osd_displayport.jpg
+   :target: ../_images/msp_osd_displayport.jpg
+
+Features
+--------
+
+DisplayPort OSDs can render all the panel items supported by the ArduPilot's onboard OSD.
+Features such as multiple screen switching, multiple units and statistics are supported as well, please refer to the :ref:`onboard OSD documentation <common-osd-overview>`  for more info.
+
+By setting :ref:`MSP_OPTIONS<MSP_OPTIONS>` bit 3 to 1 one can force ArduPilot to impersonate betaflight and use a betaflight compatible font table on the remote OSD system.
+This is required if the remote OSD system doesn not have an ArduPilot compatible fonts table. MWOSD already supports custom fonts and therefore does not require this hack while FatShark's SharkByte will support custom fonts in a future release.
+Default behaviour is to use the ArduPilot fonts table.
+
+Configuration
+-------------
+
+To enable MSP DisplayPort OSD, set the following parameters (using SERIAL port 2 as the port to attach to the Air unit using both TX and RX lines):
+
+ - :ref:`OSD_TYPE<OSD_TYPE>` = 5
+ - :ref:`SERIAL2_PROTOCOL<SERIAL2_PROTOCOL>` = 42
+
 Testing OSD with SITL
```

Comment:
It might be worth noting that DisplayPort functionality cannot yet be tested with SITL, we actually test only the DJI backend